broker-daemon/index.js

```diff
@@ -170,18 +186,11 @@ class BrokerDaemon {
           this.logger.info(`Initializing BlockOrderWorker`)
           await this.blockOrderWorker.initialize()
           this.logger.info('BlockOrderWorker initialized')
-        })(),
-        ...Array.from(this.engines, async ([ symbol, engine ]) => {
-          this.logger.info(`Validating engine configuration for ${symbol}`)
-          try {
-            await engine.validateNodeConfig()
-          } catch (e) {
-            throw new Error(`Engine for ${symbol} failed to validate: ${e.message || e.details}`)
-          }
-          this.logger.info(`Validated engine configuration for ${symbol}`)
-        })
+        })()
       ])
 
+      this.validateEngines()
```

Comment:
I would make a note here that this will run in the background until the engines are validated, otherwise it's a bit weird to have a side-effects only call like this